src/components/menuChooser.jsx

```diff
@@ -1,31 +1,60 @@
+import React from "react";
 import { useState, useEffect } from "react";
-import { useLocation } from "react-router-dom";
-import configurations from "../config";
+import { useNavigate } from "react-router-dom";
+import useSessionStorage from "../hooks/useSessionStorage";
+import NormalizationImg from "../logos/Normalization-img.svg";
+import SizingAppImg from "../logos/SizingApp-img.svg";
+import ChoosingAppArrow from "../logos/ChoosingApp-arrow.svg";
+import MenuChooserTail from "./MenuChooserAppDeatils";
+const {
+  VITE_SIZING_APP_GROUP_ID: SIZING_APP_GROUP_ID,
+  VITE_DATA_NORMALIZATION_GROUP_ID: DATA_NORMALIZATION_GROUP_ID
+} = process.env;
 
 function MenuChooser() {
-  const { search } = useLocation();
-  const [memberGroups, setGroups] = useState([]);
-  const { azureUserGroups } = configurations[process.env.NODE_ENV];
+  const navigate = useNavigate();
+  const [canAccessSizingApp, setAccessSizingApp] = useState(false);
+  const [canAccessWorkBenchApp, setAccessWorkBenchApp] = useState(false);
+  const [groups] = useSessionStorage("userGroups");
 
   useEffect(() => {
-    if (azureUserGroups && search) {
-      const groups = new URLSearchParams(search).get("groups").split(",");
-      const groupsToAccess = groups
-        .filter((groupId) => !!azureUserGroups[groupId])
-        .map((id) => azureUserGroups[id]);
-      if (groupsToAccess) {
-        setGroups(groupsToAccess);
-      }
-    }
-  }, [azureUserGroups, search]);
+    setAccessSizingApp(groups?.includes(SIZING_APP_GROUP_ID));
+    setAccessWorkBenchApp(groups?.includes(DATA_NORMALIZATION_GROUP_ID));
+  }, [groups, setAccessSizingApp, setAccessWorkBenchApp]);
 
   return (
-    <div>
-      {memberGroups.map((e) => (
-        <p key={e}>{e}</p>
-      ))}
+    <div className="text-center h-screen bg-bg">
+      <h1 className="text-xs font-bold text-black pt-10">
+        Choose the Application
+      </h1>
+      
+      {canAccessWorkBenchApp && canAccessSizingApp && (
+        <p className="text-sm mt-4">
+          You Have Permission to access Data Normalization and <br />
+          Sizing app. You can select one from below
+        </p>
+      )}
+
+      <div className="flex flex-row justify-center items-center">
+        {canAccessWorkBenchApp && (
+          <MenuChooserTail
```

Comment:
Why am I still seeing `tail` in a component name ?

Reply:
addressed